code/modules/atmospherics/machinery/pipes/multiz.dm

```diff
@@ -48,13 +48,14 @@
 
 ///Attempts to locate a multiz pipe that's above us, if it finds one it merges us into its pipenet
 /obj/machinery/atmospherics/pipe/multiz/pipeline_expansion()
-	var/turf/local_turf = get_turf(src)
-	for(var/obj/machinery/atmospherics/pipe/multiz/above in SSmapping.get_turf_above(local_turf))
+	for(var/obj/machinery/atmospherics/pipe/multiz/above in SSmapping.get_turf_above(associated_loc))
 		if(!is_connectable(above, piping_layer))
 			continue
+
 		nodes += above
 		above.nodes += src //Two way travel :)
-	for(var/obj/machinery/atmospherics/pipe/multiz/below in SSmapping.get_turf_below(local_turf))
+
+	for(var/obj/machinery/atmospherics/pipe/multiz/below in SSmapping.get_turf_below(associated_loc))
```

Comment:
don't you need to use nullspace_contents here

Reply:
yes